front/components/dialogs/shareIdeaByEmail.vue

```diff
@@ -26,6 +26,7 @@
     <section class="modalBody">
       <form id="share-idea-without-login-form" @submit.stop.prevent="sendEmail">
         <v-text-field
+          v-show="form.name === ''"
```

Comment:
if we want to hide something for logged in user we shouldn't check for names, we should check if user loggedin or not, there a few places in code where we do that  e.g. $store.getters['cognito/isLoggedIn']